Wendy/assignment1Answers.html

```diff
@@ -0,0 +1,219 @@
+					<li> (Math.abs((Math.abs(t-b))-(Math.abs(l-r))))*Math.PI; </li?
+				</ul>
+				</div>
+				<div class="col-md-6">
+				<h3> d.  The circle enclosing the rectangle</h3>
+				<ul>
+					<li> Step 1:  Determine the length of the longest side.</li>
+					<li> Step 2:  Multiply that number by Math.PI</li>
+					<li> ((l-r)<(t-b)&&((t-b)*Math.PI))||((t-b)<(l-r)&&((l-r)*Math.PI));</li>
+				</ul>
+				<h3> e.  Variables for expressing smaller rectangles</h3>
+				<ul>
+					<li> var areaSmall</li>
+					<li> 1. ((l-r)/3)*((t-b)/3) </li>
+					<li> var positionT</li>
+					<li> 2. (t-b)/3 + b + (t-b)/3;;</li>
+					<li> varSquare (boolean checking if the small rec is a square)
+					<li> 3. (Math.abs(t-b))===(Math.abs(r-l))</li>
+					<li> var positionR</li>
+					<li> 4. (l-r)/3 + r + (l-r)/3;
+
+					<!-- <li> var areaSmall (to represent the center small rectangle)</li>
+					<li> 1. areaSmall = (t/3)*(l/3)</li>
+					<li> var tallSmall (to represent the tallest side)</li>
+					<li> 2. var tallSmall = (l>=t)&&(l/3)||!(l>=t)&&(t/3)</li>
+					<li> var shortSmall (to reperesent the shorter side)</li>
+					<li> 3. var shortSmall = (l<=t)&&(l/3)||(t<=l)&&(t/3)</li>
+					<li> var square (boolean checking if the small rectangle is a square)
+					</li> 4. varSquare = ((t-l) === 0)</li> -->
+				</ul>
+				</div>
+			</div>	
+			
+			<div class="references">
+			<div class="row">
+			<h3>Resources used</h3>
+				<div class="col-md-12">
+				<ul>
+					<li><a href="http://www.w3schools.com/jsref/jsref_pi.asp"> Math.PI info</a></li>
+					<li><a href="http://www.w3schools.com/js/js_variables.asp"> Specific Definitions for Variables in JavaScript</a></li>
+				</div>
+			</div>
+			</div>		
+			<div class="row" id ="part4">
+			<h3>Part IV</h3>
+			<h3> Checker Board solving for Number, Row and Column</h3>
+				<div class="col-md-6">
+					
+					<ul>
+						<li> a.) n = ((r*7)+(r+c))</li>
+						<li> b.) r = ((n-8)%8)</li>						
+						<li> c.) c = (n%8)</li>
+						<li> d.) if((r+c)%2===0){
+							'black';
+						} else {
+							'white';
+						}
+						</li>
+					</ul>
+				</div>
+			
+				<div class="col-md-6">
+					<img src="img/checkerBoard.jpg"></img>
+				</div>
+			</div>
+			
+			<div class="row" id ="part5">
+			<h3>Part V</h3>
+			<h3>Representation of a fraction (n/d) with 2 integer variables.</h3>
+				<div class="col-md-6">
+				<h3> Solving using Math.floor</h3>
+					<ul>
+						<li> var n is the numerator var d is the denominator</li>
+						<li> var a = Math.floor(n/d);</li>
+						<li> var b = (n -(d*a));</li>
+						<li> n = b;</li>
+						<lI> a + ' ' + b + '/' + d; </li>
+					</ul>
+				</div>
+				<div class="col-md-6">
+				<h3>Solving using other magical means such as modulo</h3>
+					<ul>
+						<li> var Big = n%d; </lid>
+						<li> var Small = (n -(d*b)); </li>
+						<li> Big + ' ' + Small + '/' + d; </li>
+					</ul>
+				</div>
+			</div>
+						
+				
+			
+
+
+
+</div>
+</body>
```

Comment:
Hey Wendy, great job on this homework. And nice work on the formatting, real fancy! Moving forward, for grading purposes, can you include the answers in a plain .js or .md format? I like to run your code and it is hard to pull it out of the html document.